scripts/management/query_mgmt_db.py

```diff
@@ -31,22 +73,97 @@ def print_run_status(db, run_name, error=False, count=False, config_file=None):
                 )
             )
     elif count:
-        vals = []
-        for i in range(1, 7):
-            vals.append(
-                db.execute(
-                    """SELECT COUNT(*)
-                        FROM state
-                        WHERE status = ?
-                        """,
-                    (i,),
-                ).fetchone()[0]
-            )
-        print(
-            "created: {}, queued: {}, running: {}, completed: {}, failed: {}, other: {}, total: {}".format(
-                *vals, sum(vals)
-            )
-        )
+        if detailed_count:
```

Comment:
This block is a behemoth. I'm guessing it could be simplified / functioned out? Also, maybe it might be a better fit in a separate script?